### PR TITLE
LPS-158558 [Raylife AP]  Create form layout

### PR DESCRIPTION
**Pull request by edbbarros**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+type hasRequiredErrorTypes = {
+  bodilyInjury: boolean,
+  propertyDamage: boolean,
+  underinsuredInjured: boolean,
+  uninsuredDamage: boolean,
+  medical: boolean,
+  comprehensive: boolean,
+  collision: boolean
+};
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:27]: Object keys should be in natural ascending order and "type" objects should end with `; `.
```suggestion
type hasRequiredErrorTypes = {
                bodilyInjury: boolean;
		collision: boolean;
		comprehensive: boolean;
		medical: boolean;
		propertyDamage: boolean;
		underinsuredInjured: boolean;
		uninsuredDamage: boolean;
};
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+type HasValidationsTypes = {
+  bodilyInjury: boolean,
+  propertyDamage: boolean,
+  underinsuredInjured: boolean,
+  uninsuredDamage: boolean,
+  medical: boolean,
+  comprehensive: boolean,
+  collision: boolean
+}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:37]: Same here.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+
+const hasValidations: HasValidationsTypes = {
+  bodilyInjury: false,
+  propertyDamage: false,
+  underinsuredInjured: false,
+  uninsuredDamage: false,
+  medical: false,
+  comprehensive: false,
+  collision: false
+}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:58]: Object keys should be sorted in natural ascending order.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+
+const CoverageInfoForm = () => {
+
+  return (
+    <div className='container p-4 m-0'>
+      <p className="font-weight-bolder mb-4 mt-0 text-paragraph-sm text-uppercase">CHOOSE YOUR COVERAGES</p>
+      <hr />
+      <div className="font-weight-bolder mb-4 mt-0 text-paragraph-sm text-uppercase">
+        <u>POLICY COVERAGES</u>
+      </div>
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:69]:
```suggestion
      <div className="font-weight-bolder mb-4 mt-0 text-paragraph-sm text-uppercase">
        <u>Policy Coverages</u>
      </div>
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+    label: 'CHOOSE AN OPTION',
+    value: 'CHOOSE AN OPTION'
+  },
+  {
+    label: '$15,000 / $30,000',
+    value: '15,000 / $30,000'
+  },
+  {
+    label: '$30,000 / $60,000',
+    value: '30,000 / $60,000'
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:30]:
```suggestion
    label: '$15,000 / $30,000',
    value: '$15,000 / $30,000'
  },
  {
    label: '$30,000 / $60,000',
    value: '$30,000 / $60,000'
```

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx (hunk 1 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:27]: I will remove it

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx (hunk 3 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:58]: I will remove it

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx (hunk 2 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:37]: Done!

**Review by joanagrego — COMMENTED** [review]
Nice job!  :smiley: :+1:

**Review by joanagrego — APPROVED** [review]
Good job! :smiley: :+1:

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties [hunk 6]
```diff
@@ -13425,7 +13425,7 @@ scheduled-events=Antolaturiko gertaerak
 scheduled-jobs=Scheduled Jobs (Automatic Copy)
 scheduled-publish-layout-local=Scheduled Local Publish (Automatic Copy)
 scheduled-publish-layout-remote=Scheduled Remote Publish (Automatic Copy)
-scheduled-start-date=Scheduled Start Date (Automatic Copy)
+scheduled-start-date=Programatutako hasierako data
```
Comment (eliasantosf) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties:13428]: These Language files should not have been committed, it needs to be removed.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+import classNames from 'classnames';
+
+import { bodilyInjuryOptions, collisionOptions, comprehensiveOptions, medicalOptions, propertyDamageOptions, underinsuredInjuredOptions, uninsuredDamageOptions } from './SelectOptions';
+
+
+const CoverageInfoForm = () => {
+
+  return (
+    <div className='container m-0 p-4'>
+      <p className="font-weight-bolder mb-4 mt-0 text-paragraph-sm text-uppercase">CHOOSE YOUR COVERAGES</p>
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:25]: The sentence is not capitalized.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+          </label>
+        </div>
+
+        <div className='col-sm'>
+
+        </div>
+      </div>
+
+      <div className="font-weight-bolder mb-4 mt-3 text-paragraph-sm text-uppercase">
+        <u>VEHICLE 1: 2018 Toyota</u>
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:164]: The sentence is not capitalized.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+ */
+
+export const bodilyInjuryOptions = [
+  {
+    label: '',
+    value: ''
+  },
+  {
+    label: 'CHOOSE AN OPTION',
+    value: 'CHOOSE AN OPTION'
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:22]: You can use Camel Case here.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts [hunk 10]
```diff
@@ -0,0 +1,169 @@
...
+
+];
+export const propertyDamageOptions = [
+  {
+    label: '',
+    value: ''
+  },
+  {
+    label: 'CHOOSE AN OPTION',
+    value: 'CHOOSE AN OPTION'
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:45]: You can use Camel Case here.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts [hunk 11]
```diff
@@ -0,0 +1,169 @@
...
+  }
+];
+export const underinsuredInjuredOptions = [
+  {
+    label: '',
+    value: ''
+  },
+  {
+    label: 'CHOOSE AN OPTION',
+    value: 'CHOOSE AN OPTION'
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:67]: You can use Camel Case here.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+  }
+];
+export const uninsuredDamageOptions = [
+  {
+    label: '',
+    value: ''
+  },
+  {
+    label: 'CHOOSE AN OPTION',
+    value: 'CHOOSE AN OPTION'
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:89]: You can use Camel Case here.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts [hunk 13]
```diff
@@ -0,0 +1,169 @@
...
+  }
+];
+export const medicalOptions = [
+  {
+    label: '',
+    value: ''
+  },
+  {
+    label: 'CHOOSE AN OPTION',
+    value: 'CHOOSE AN OPTION'
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:111]: You can use Camel Case here.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts [hunk 14]
```diff
@@ -0,0 +1,169 @@
...
+  }
+];
+export const comprehensiveOptions = [
+  {
+    label: '',
+    value: ''
+  },
+  {
+    label: 'CHOOSE AN OPTION',
+    value: 'CHOOSE AN OPTION'
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:133]: You can use Camel Case here.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+  }
+];
+export const collisionOptions = [
+  {
+    label: '',
+    value: ''
+  },
+  {
+    label: 'CHOOSE AN OPTION',
+    value: 'CHOOSE AN OPTION'
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:155]: You can use Camel Case here.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/NewApplication/index.tsx [hunk 16]
```diff
@@ -30,6 +31,9 @@ const NewApplication = () => {
 			{state.currentStep === 1 && <VehicleInfo />}
 
 			{state.currentStep === 2 && <DriverInfo />}
+
+			{state.currentStep === 3 && <Coverage />}
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/NewApplication/index.tsx:35]: Should your step be 4?

**Review by eliasantosf — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/NewApplication/index.tsx (hunk 16 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/NewApplication/index.tsx:35]: Done.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx (hunk 7 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:25]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx (hunk 8 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:164]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts (hunk 9 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:22]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts (hunk 10 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:45]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts (hunk 12 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:89]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts (hunk 13 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:111]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts (hunk 14 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:133]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts (hunk 15 above)
Comment (edbbarros) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/SelectOptions.ts:155]: Done!

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties (hunk 6 above)
Comment (edbbarros) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties:13428]: Done!

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @edbbarros

**Comment by edbbarros** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by edbbarros** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:forward

**Pull request closed without merging**